### ISSUE TITLE
vegetarian/vegan quest : add "limited" value

### ISSUE THREAD

**Issue by cbobinec** [streetcomplete/StreetComplete#6013]
["limited" value is now available for diet:vegan and vegetarian](https://wiki.openstreetmap.org/w/index.php?title=Key:diet:vegan) : it would be nice to add this answer ?

**Comment by matkoniecz** [comment]
no, as it is useless

any place that has salt and pepper available passes "The establishment offers very few vegan products. "

**Comment by betalars** [comment]
@matkoniecz excuse me, how is this useless?
> The establishment does not clearly indicate vegan products but offers some options. For example, restaurants that do not offer a comparable choice of full vegan meals but can arrange for a limited vegan option, e.g. by offering an arrangement of vegan side dishes and/or excluding non-vegan ingredients.

Okay I do not understand at all why limited per this definition is useless.

Many places will have no vegan options whatsoever.

And if anything, the comparable tag is useless in practice. Restaurants seem to either not (really) care or go full vegan, at least where I've been. Excluding Asian takeout I can think of no more than 3 off the top of my head that would meet the comparable vegan tag and I can probably name 15-20 fully vegan restaurants.

However a bunch of place do have limited options. This includes kebab places that have falafel and and one vegan sauce. This includes your average fast-food chain that has managed to introduce one vegan option. This includes a bunch of small restaurants that can't offer a vegan option but will go out of their way to be a good host when you ask for it. This includes the average cafe that has no vegan cake at all but does at least offer oat milk.

And maybe there are malicious users that would consider salt and pepper or a side of french fries to be a limited vegan option. But come on, this is not what that tag nor "vegan" means.

See also vegan tomatoes and asbestos-free cereal in tom scotts video on language paradigms.

**Comment by matkoniecz** [comment]
> This includes kebab places that have falafel and and one vegan sauce. This includes a bunch of small restaurants that can't offer a vegan option but will go out of their way to be a good host when you ask for it. This includes the average cafe that has no vegan cake at all but does at least offer oat milk.

as far as I know that falls under `yes`

if someone wished or wanted for `yes` actually means "it has wide selection of vegan products" then it should use a different value, but it seems to late to select a different one

**Comment by mnalis** [comment]
> or a side of french fries to be a limited vegan option. 

My kid would find that a perfectly valid vegan menu (as far as he's concerned it could be even classified as full `diet:vegan=yes` 😸 ).


And that is the issue: what is _obviously wrong_ tagging for you is _perfectly valid_ tagging for someone else, and this breaks  [verifiability](https://wiki.openstreetmap.org/wiki/Verifiability), making it problematic in OSM generally. Root cause is that meaning of `diet:vegan=limited` was under-specified and unclear, and that leads to it failing basic [StreetComplete tenets](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md):

- _👨‍💻 Users are no experts: No knowledge about OpenStreetMap or any other background knowledge must be necessary_ (i.e. people who are not Vegans themselves nor know details of Veganism should be able to answer the Quest without any qualms).
- _🐿️ Easy answer: Users are out and about and impatient. A quick, straightforward and clear answer must be possible_

So, IMHO, only way to answer that quest correctly by StreetComplete user is if it is **labeled properly**[^1] by restaurant owner. And that covers `yes` and `only` answers (if _some_ / _all_ meals are labeled as Vegan), and `no` (if _none_ of the meals are labeled as Vegan).

> But come on, this is not what that tag nor "vegan" means.

Isn't it?

[diet:vegan OSM wiki](https://wiki.openstreetmap.org/wiki/Key:diet:vegan) says (emphasis mine):

> Describes if an establishment offers vegan products. According to the [Vegan Society definition of veganism](https://www.vegansociety.com/go-vegan/definition-veganism), it "is a way of living which seeks to exclude, as far as is possible and practicable, **all forms of exploitation of, and cruelty to, animals** for food, clothing **or any other purpose**." In practice, this definition encompasses animal meat, seafood, dairy, eggs, honey, gelatin, leather, wool, products tested on animals and any other product which **verifiably** and unnecessarily had non-human **animals harmed in the process**.

[Wikipedia](https://en.wikipedia.org/wiki/Veganism) also says:

> Veganism excludes all forms of animal use, whether in agriculture for labour or food, in clothing and industry, in entertainment, or in services. 

So, even if your falafel from that example does not contain any meat/diary/egg/etc ingredients, **if** e.g. beast of burden was used to power wheel for grounding those beans, or for transporting the ingredients, that falafel end-product is **not** Vegan. Just as your [wine is not vegan](https://www.allrecipes.com/article/why-are-some-wines-not-vegan/) if incompatible fining agents (like casein or gelatin or chitosan etc.) were used during its production.  

---

**TL;DR:** being Vegan is _**much more**_ than _"being vegetarian and also not eating cheese and honey"_. 

**Without the _labeling_,  one can not really say** if that soy steak is **vegan** just by looking at it or tasting it; just like one cannot tell if that chicken nugget is [Kosher](https://en.wikipedia.org/wiki/Kosher_foods#Ritual_slaughter) or [Halal](https://en.wikipedia.org/wiki/Halal#Halal_meat) just by looking at it or tasting it.


[^1]:  hopefully truthfully; but trustful certifications are all another bag of beans

**Comment by betalars** [comment]
> as far as I know that falls under `yes`

The official wiki says:

>  restaurants that clearly indicate and offer a comparable choice of vegan meals

Getting to eat falafel kebap with only spicy sauce is not comparable. This is actually infuriating and you get tired of it really fast.

I consider a falafel place to offer a **comparable** vegan option if it i has
 - at least one meat-alternative
 - at least one vegan cream-sauce
 - at least one cheese-alternative when they also have pizza on the menu (most places in my area tend to also serve pizza)

Because then I can kind of eat at that place like a person that isn't vegan. And as non-vegans do not need to ask the chef to create them a special meal, I also do not consider restaurants that do their best when you ask them nicely to meet the "yes" label.

Hence I like "limited" a lot. Because those places are still a lot better than those that just have nothing for me to eat.

**Comment by betalars** [comment]
> Isn't it?

Of course potatoes and tomatoes are _technically_ vegan. They are also asbestos-free. But putting a vegan label on a potato is as weird as putting an asbestos-free label on a potato.

But just let me classify some examples of how I understand the tags:

vegan:no: There are no options labelled as vegan and even when you look for yourself on the menu you couldn't eat anything other than maybe a salad or some fries. No vegan would want to know about this place when planning a trip.

vegan:limited: A vegan would consider this place when going on a trip, but there's a catch. Maybe there's only one vegan option. Maybe you'd have to ask for accommodation. Maybe some options that have no animal products are included on the menu but they are not labelled.

vegan:yes: You'd consider taking a vegan on a date to this place. There are comparable vegan options and they are labelled as such. A vegan person will have a similar experience as a non-vegan when going to this place.

began:only ... I think this one explains itself.

**Comment by matkoniecz** [comment]
if you want to invent new tags or redefine values of established tags then SC issue tracker is not a place for it

see https://taghistory.raifer.tech/#***/diet%3Avegan/limited https://taghistory.raifer.tech/#***/diet%3Avegan/yes&***/diet%3Avegan/limited

**Comment by mnalis** [comment]
I've already noted why I think `diet:vegan=limited` is a bad tag. It is not because it is inherently bad, but it is because it was poorly defined and thus [unverifiable](https://wiki.openstreetmap.org/wiki/Verifiability), and it is way too late now to try to change its meaning[^1].   Any attempt to fix it at this point in time would only increase confusion (e.g. was that `limited` according to old definition, or according to new one?)

Of 3 persons discussing it in this thread, we already have 4 different definitions of what it means. That is IMHO highly incompatible with StreetComplete quest guidelines mentioned [above](https://github.com/streetcomplete/StreetComplete/issues/6013#issuecomment-2869116560).




[^1]: because you would first have to contact all data consumer and all mappers who have ever used them, remove all existing instances of `diet:vegan=limited` from the map, and convince all mappers to retag according to new standards. That would be, to say the least, extremely hard. Perhaps a [new tag value](https://wiki.openstreetmap.org/wiki/Proposal), with good description and a dozen examples of what is covered by it, and what is not, might work. But `limited` is (unfortunately) burned.

**Comment by betalars** [comment]
Okay I concede this tag is virtually never used and I seem to  be the only one that finds it useful.

I'll guess I will disable these quests and not trust osm when it comes to vegan restaurants until I had a chance to discuss this with the community..

**Comment by betalars** [comment]
> One cannot separate (and ignore) different parts of being Vegan and still call it Veganism, without bastardizing the term.

I will note that there are different people tho.

I'm mainly vegan because having my energy go trough an entire cow before I eat it just wastes tons of energy and land so I do my best to be friendly to the environment and also support environment political causes but I woudn't be mad at a date that took me to a `vegan:yes` place that has an actually comparable selection even if they decide to eat "less energy efficient" than I am.

But I know I am not the majority of vegans so maybe this is yet another reason why that limited tag seems more useful to me than to a lot of others.

Anyways I am now going to unsubscribe from this issue.
